command/sub8_missions/tools/waypoint_utility.py

```diff
@@ -24,12 +27,32 @@ def __init__(self):
         When determining left and right, the wire faces away from you'''
         rospy.init_node('spacenav_positioning')
 
+        if len(sys.argv) == 0:
```

Comment:
You can set a default in the launch file. If we want to add more driving modes later that might help, but it doesn't really matter now since you've already done it.